package.json

```diff
@@ -6,6 +6,8 @@
     "dev": "next dev",
     "build": "next build",
     "start": "next start",
+    "start:with-cron-jobs": "run-p start cron",
```

Comment:
It seems that `run-p` is not listed in the dependencies

Reply:
Non-intuitively, `run-p` is provided by `npm-run-all`.